docs/admin/networking/index.md

```diff
@@ -77,6 +77,64 @@ direct.
 In general, [port forwarded](./port-forwarding.md) web apps are faster than
 dashboard-accessed web apps.
 
+## Latency
+
+Coder measures and reports several types of latency, providing insights into the performance of your deployment. Understanding these metrics can help you diagnose issues and optimize the user experience.
+
+There are three main types of latency metrics for your Coder deployment:
+
+- Dashboard-to-server latency:
+  
+  The Coder UI measures round-trip time to the Coder server using the browser's Performance API.
+  
+  This appears in the user interface next to your username, showing how responsive the dashboard is.
```

Comment:
It's a bit more complex if there are multiple workspace proxies. The dashboard is always served from the primary Coder server deployment, which will be one specific entry, although I don't think we do anything special to indicate which one, unfortunately.

The other workspace proxies serve applications from the workspace, like web IDEs, the web terminal, etc. A low ping there will help those applications be more responsive, but it only measures part of the round-trip. The proxy to workspace leg is not measured.

Reply:
I had trouble figuring out how to add that to this doc in a way that doesn't take it too far off course, so I added it to `workspace-proxies.md` in https://github.com/coder/coder/pull/16734/commits/9e7dc7b1d01e0c27c7cf1ff4cb331f13e747a119

I also force-pushed to this branch because I repointed it to `main` so that the latency info isn't tied to a rework of the networking docs in general. hopefully I did that as cleanly as possible 🤞